frontend/src/routes/(authenticated)/admin/ProjectTable.svelte

```diff
@@ -53,6 +66,18 @@ async function softDeleteProject(project: Project): Promise<void> {
         notifyWarning($t('delete_project_modal.success', {name: project.name, code: project.code}));
     }
 }
+
+  const migrationStatusTable = {
+    [ProjectMigrationStatus.Migrated]: 'i-mdi-checkbox-marked-circle-outline',
+    [ProjectMigrationStatus.Migrating]: 'loading loading-spinner loading-xs',
+    [ProjectMigrationStatus.Unknown]: 'i-mdi-help-circle-outline',
+    [ProjectMigrationStatus.PrivateRedmine]: 'i-mdi-checkbox-blank-circle-outline',
+    [ProjectMigrationStatus.PublicRedmine]: 'i-mdi-checkbox-blank-circle-outline',
+  } satisfies Record<ProjectMigrationStatus, string>;
+  function migrationStatusIcon(migrationStatus?: ProjectMigrationStatus): string {
+    migrationStatus = migrationStatus ?? ProjectMigrationStatus.Unknown;
+    return migrationStatusTable[migrationStatus] ?? migrationStatusTable[ProjectMigrationStatus.Unknown];
```

Comment:
The fallbacks here (i.e. `??`) are redundant. How about this:
```suggestion
    return migrationStatusTable[migrationStatus ?? ProjectMigrationStatus.Unknown];
```

Reply:
Just deleting line 78 would be better. We do need the ` ?? migrationStatusTable[ProjectMigrationStatus.Unknown]` part, in case we later add a new migration status that isn't in the table. But given that we look up the icon for Unknown if the value wasn't found, it's safe to skip the part where a value is assigned to `migrationStatus` if it was undefined.

Reply:
Actually if I just delete line 78 then Typescript complains that the type `undefined` isn't a valid index into the `migrationStatusToIcon` record type, so both are needed and this isn't actually redundant. Rolling back my change (by removing commit a012747) and will leave this one as-is.